crypto/rand/randfile.c

```diff
@@ -280,8 +280,16 @@ const char *RAND_file_name(char *buf, size_t size)
     struct stat sb;
 #endif
 
+    /* Find the directory where the state file should go. */
+#ifdef OPENSSL_SYS_WINDOWS
+    if ((s = getenv("RANDFILE")) == NULL) {
+        s = getenv("TEMP");
+        if (s == NULL)
+            s = getenv("APPDATA");
```

Comment:
Correct. But why expose ourselves to that risk when we don't have to? Don't put it in AppData\Roaming in the first place and it won't be a problem.